build.sbt

```diff
@@ -343,22 +343,24 @@ componentArtifacts := {
     (flinkKafkaComponents / assembly).value -> "components/flink/flinkKafka.jar",
     (liteBaseComponents / assembly).value -> "components/lite/liteBase.jar",
     (liteKafkaComponents / assembly).value -> "components/lite/liteKafka.jar",
+    (liteRequestResponseComponents / assembly).value -> "components/lite/liteRequestResponse.jar",
     (openapiComponents / assembly).value -> "components/openapi.jar",
     (sqlComponents / assembly).value -> "components/sql.jar"
   )
 }
 
 lazy val modelArtifacts = taskKey[List[(File, String)]]("model artifacts")
 modelArtifacts := {
-  ((generic / assembly).value-> "model/genericModel.jar") :: Nil
+  List(
+    (defaultModel / assembly).value-> "model/defaultModel.jar",
+    (flinkExecutor / assembly).value-> "model/flinkExecutor.jar",
+  )
 }
 
 lazy val devModelArtifacts = taskKey[List[(File, String)]]("dev model artifacts")
 devModelArtifacts := {
   modelArtifacts.value ++ List(
-    (flinkManagementSample / assembly).value -> "model/managementSample.jar",
-    (requestResponseSample / assembly).value ->  "model/requestResponseSample.jar",
-    (liteModel / assembly).value -> "model/liteModel.jar"
+    (flinkManagementSample / assembly).value -> "model/managementSample.jar"
```

Comment:
maybe we could change also this artifact to `devModel,jar`. It looks like it is the only thing to close cleanups in model names

Reply:
+ changes in docs...

Reply:
done... renamed all references I could find...